### DISCUSSION TITLE
The zoom window dissappearing from the toolbar for some pdfs

### DISCUSSION THREAD

**Discussion by wkusmierczuk** [fengyuanchen/viewerjs#544] [Q&A] [answered]
Hello,
I noticed that for some random pdf documents, the zoom window doesn't show up at all. This seems to be triggered on horizontally aligned documents more often. Is this a known issue? What is the cause of this behavior?
![image](https://user-images.githubusercontent.com/87985685/164016598-1c98b813-9fb3-45be-8106-bf7a1c3e0836.png)

**Comment by fengyuanchen** [comment, marked as answer]
Maybe you are talking about another library: https://github.com/webodf/ViewerJS

**Reply by wkusmierczuk** [reply]
You're right, somehow I mixed up those repositories :-) Sorry for confusion. Thanks
